virtwho/config.py

```diff
@@ -1236,3 +1279,8 @@ def get_effective_config():
     # We don't have one, let us make one
     _effective_config = EffectiveConfig()
     return _effective_config  # Initialize an EffectiveConfig with no options
+
+
+def reset_effective_config():
+    global _effective_config
+    _effective_config = None
```

Comment:
BTW: I would modify `EffectiveConfig` to be singleton. Using `global` is not my favorite approach. ;-)